adapters/eplanning/eplanning.go

```diff
@@ -114,6 +114,13 @@ func (adapter *EPlanningAdapter) MakeRequests(request *openrtb.BidRequest, reqIn
 	var pageDomain string
 	if request.Site != nil && request.Site.Domain != "" {
 		pageDomain = request.Site.Domain
+	} else if request.Site != nil && request.Site.Page != "" {
```

Comment:
Nitpick: The `request.Site != nil` check is repeated. What do you think of pushing down the `Site.Domain` and `Site.Page` check within a single nil check?